### PR TITLE
Code clean

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: syncd/AsicView.cpp [hunk 1]
```diff
@@ -829,7 +829,7 @@ void AsicView::asicRemoveObject(
                 break;
 
             case SAI_OBJECT_TYPE_NAT_ENTRY:
-                m_soNatEntries[currentObj->m_str_object_id] = currentObj;
+                m_soNatEntries.erase(currentObj->m_str_object_id);
```
Comment (jleveque) [inline syncd/AsicView.cpp:832]: Was this a bug?

Comment (kcudnik) [reply syncd/AsicView.cpp:832]: yes :)

Comment (jleveque) [reply syncd/AsicView.cpp:832]: Good find!

**Review by jleveque — APPROVED** [review]
LGTM.

**Pull request merged**